src/Resources/config/services/controller.xml

```diff
@@ -117,7 +117,6 @@
         <service id="Sylius\PayPalPlugin\Controller\UpdatePayPalOrderAction">
             <argument type="service" id="Sylius\PayPalPlugin\Provider\PaymentProviderInterface" />
             <argument type="service" id="Sylius\PayPalPlugin\Api\CacheAuthorizeClientApiInterface" />
-            <argument type="service" id="Sylius\PayPalPlugin\Api\OrderDetailsApiInterface" />
```

Comment:
It is not a BC Break, as it is excluded in our BC Policy, but possibly it would be nice to add a note to the UPGRADE file with such a change

Reply:
Done

Reply:
Thank you for that, I've moved it to the proper version yet 🖖🏻